API/Endpoints/SessionEndpoints.cs

```diff
@@ -80,7 +80,7 @@ async Task<Results<Ok<List<GetSessionsResponseDto>>, NotFound, BadRequest>> (Cla
 
             return TypedResults.Ok(result.Value);
             
-        }).RequireAuthorization(nameof(Roles.AnonymousUser));
+        }).RequireAuthorization(Policies.AllowAllRoles);
```

Comment:
Does this mean that all users can access a session now instead of just anonymous users?
![image](https://github.com/user-attachments/assets/e6d8396e-fa9c-4fa9-8bb2-a02edbf95de1)

Reply:
No you have to look at the changes for the policy.

Reply:
Please elaborate